frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeSidebar.tsx

```diff
@@ -163,17 +163,9 @@ function getQuery(query: Lib.Query, isDefaultAggregationRemoved: boolean) {
   const shouldAddDefaultAggregation =
     !hasAggregations && !isDefaultAggregationRemoved;
 
-  const operator = Lib.availableAggregationOperators(query, STAGE_INDEX).find(
-    operator => {
-      const { shortName } = Lib.displayInfo(query, STAGE_INDEX, operator);
-      return shortName === "count";
-    },
-  );
-
-  if (operator && shouldAddDefaultAggregation) {
-    const clause = Lib.aggregationClause(operator);
-    return Lib.aggregate(query, STAGE_INDEX, clause);
+  if (!shouldAddDefaultAggregation) {
```

Comment:
could you please explain why this change is introduced?

Reply:
I reused the new `Lib.aggregateByCount`, the function should work the same as before.
I "inverted" the condition to have as little code in `if` block as possible.